play-json/jvm/src/test/scala-3/play/api/libs/json/JsonMemoryFootprintSpec.scala

```diff
@@ -46,7 +46,7 @@ class JsonMemoryFootprintSpec extends AnyFreeSpec {
     "obj0" in assertSizes(arr1KB("{}"), 7432, 7432)
     "obj1" in assertSizes(arr1KB("""{"a":6}"""), 29568, 31568)
     "nums" in assertSizes(arr1KB("6"), 42104, 42104)
-    "arr0" in assertSizes(arr1KB("[]"), 42064, 42064)
+    "arr0" in assertSizes(arr1KB("[]"), 15424, 15424)
```

Comment:
The memory footprint decreased between Scala 3.2.1-RC3 and RC4: https://github.com/lampepfl/dotty/compare/3.2.1-RC3...3.2.1-RC4
(yes I did track this down). Seems like bumping the stdlib from [2.13.8 to 2.13.10](https://github.com/scala/scala/compare/v2.13.8...v2.13.10) had some performance improvements positively effecting memory usage. I am not going to track down which commit exactly caused this (https://github.com/scala/scala/commit/ef5a758e4d9e5be27ec60a2844dc966589976978 would be a candidate but not sure)